regen-network/regen-web
# feat: APP-218 add create post to SellOrdersActionsBar

## Description

https://regennetwork.atlassian.net/browse/APP-218

Added a Create Post button in the Project Page actions bar.
Refactored `ProjectDetails.DataStream` to share the `PostFlow` with `SellOrderActionsBar` in `ProjectDetails`.
Add option for Create Post button to be disabled when a project is a draft in Project Page.

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test

1. As an admin open any project page and verify that the Create Post button in the ActionsBar and the Data Stream work as expected.
2. Check that the Create Post button is disabled if the project hasn't been published (it's a draft)

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
